flexmeasures/api/v3_0/sensors.py

```diff
@@ -258,6 +270,10 @@ def trigger_schedule(  # noqa: C901
         with a target state of charge of 25 kWh at 4.00pm.
         The minimum and maximum soc are set to 10 and 25 kWh, respectively.
         Roundtrip efficiency for use in scheduling is set to 98%.
+        Aggregate consumption should be priced by sensor 9,
+        and aggregate production should be priced by sensor 10,
+        where the aggregate power flow includes the sum over sensors 13, 14 and 15.
```

Comment:
I feel we are not documenting really well yet what "inflexible-device-sensors" means. 

Here is an attempt to do better:

> where the aggregate power flow in the EMS is described by the sum over sensors 13, 14 and 15 (plus the flexible sensor being optimized, of course). 

Is this agreeable?

(if there are more inflexible sensors, this of course would not work. Maybe we should document these thoughts on a higher level, like in a tutorial. Triggering schedules is a huge part of interacting with FlexMeasures)

Reply:
Thanks for the improved description. I'll revise accordingly. In terms of documentation on a higher level, one of the follow-up steps to this PR is to "extend the documentation and the toy tutorial with scheduling within the context of a building with PV". I think that extended tutorial would be a natural place to put it.

(Why wouldn't this work if there are more inflexible sensors?)

Reply:
> Why wouldn't this work if there are more inflexible sensors?

In that case, the EMS is not fully described in the problem, so you might get weird outcomes if you only optimise a sub-part of the system. Of course, you can still do that.